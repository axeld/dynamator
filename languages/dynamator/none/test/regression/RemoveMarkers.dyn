<dynamator language="none">
  <include file="Content.dyn"/>
  <include file="RemoveMarkers.include"/>
</dynamator>
