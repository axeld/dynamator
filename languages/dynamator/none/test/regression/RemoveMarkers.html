<html>
<head>
  <title>Remove HTML 4.0 Markers</title>
</head>
<body>
<p id="removeme" align="right"><span class="test">text</span></p>
<p class="removeme"><span id="y">text</span></p>
<div id="div_1">
<p>fail</p>
</div>
<div class="div_2">
<p><span id="span_1">fail</span></p>
</div>
<div class="x div_2">
<p>fail</p>
<p>fail</p>
</div>
<div class="div_2 x">
<p>fail</p>
<p>fail</p>
</div>
<div class="x div_2 x">
<p>fail</p>
<p>fail</p>
</div>
</body>
</html>
