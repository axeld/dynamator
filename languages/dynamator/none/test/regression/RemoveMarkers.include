<dynamator>
<tag tag="*" with-attr="id">
  <attr name="id"><discard/></attr>
</tag>
<tag tag="*" with-attr="class">
  <attr name="class"><discard/></attr>
</tag>
<tag tag="div">
  <discard-tag/>
</tag>
<tag tag="span">
  <discard-tag/>
</tag>
</dynamator>