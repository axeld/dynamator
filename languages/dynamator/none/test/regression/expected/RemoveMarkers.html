<html>
  <head>
    <title>Remove HTML 4.0 Markers</title>
  </head>
  <body>
    <p align="right">text</p>
    <p>text</p>

      <p>success!</p>
    
      <p>success!</p>
    
      <p>success!</p>
    
      <p>success!</p>
    
      <p>success!</p>
      </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
